app/views/notifier/new_user.text.erb

```diff
@@ -0,0 +1,10 @@
+<%= text(".intro_text", first_name: @user.first_name, last_name: @user.last_name, login_url: new_user_session_url) %>
+
+<!-- non-standard indentation below is to prevent indents appearing in the text formatting of this email -->
```

Comment:
Not a big deal, but should we use a ruby comment instead of an html comment so it doesn't get rendered in the output.